swagger.v4.json

```diff
@@ -681,6 +681,20 @@
                         "description": "UUID for publication record",
                         "type": "string",
                         "required": true
+                    },
+                    {
+                        "name": "sort",
+                        "in": "query",
+                        "description": "Method of sorting collections. Either title or creator",
+                        "type": "string",
+                        "required": false
+                    },
+                    {
+                        "name": "perPage",
+                        "in": "query",
+                        "description": "Number of collections to load per page",
+                        "type": "integer",
+                        "required": false
```

Comment:
I think swagger supports setting a default, looks like it's 10 from a quick scan of the code!